source/MapDetailPanel.cpp

```diff
@@ -480,9 +480,10 @@ void MapDetailPanel::DrawInfo()
 						10., 10., 0., medium);
 				
 				bool hasVisited = player.HasVisited(planet);
-				font.Draw(hasVisited ? "(has been visited)" : "(not yet visited)",
+				bool isAccessible = planet->IsAccessible(player.Flagship());
```

Comment:
With the above change (only showing inaccessible planets in DrawInfo if you've visited them) this is fine, since you're guaranteed to have visited it.
If the above change is not done (DrawInfo for inaccessible planets without having visited) then this should probably be conditional on having visited the planet: ` hasVisited && planet->IsAccessible(player.Flagship());`

Reply:
I'm not sure I should draw info for inaccessible planets the player hasn't visited, considering you can't select them.